### PR TITLE
Fixed makeqr

### PR DESCRIPTION
**Pull request by Yash089610**
Also changed TEMPdir to ./

### CODE REVIEW

File: userbot/modules/qrcode.py [hunk 1]
```diff
@@ -12,11 +12,13 @@
 from asyncio import sleep
 from datetime import datetime
 from requests import post, get
-
+from telethon import events
+import asyncio
```
Comment (raphielscape) [inline userbot/modules/qrcode.py]: no full imports

**Comment by Yash089610** [comment]
@raphielscape make the necessary changes n merge

File: userbot/modules/qrcode.py [hunk 2]
```diff
@@ -12,8 +12,6 @@
 from asyncio import sleep
 from datetime import datetime
 from requests import post, get
-from telethon import events
-import asyncio
 ￼import requests
```
Comment (raphielscape) [inline userbot/modules/qrcode.py]: Do we really need a full-blown request import?

**Comment by raphielscape** [comment]
Fixed in https://github.com/RaphielGang/Telegram-UserBot/commit/33518e556169a2d921f2fbd9683968e43c0174fa , closing this PR

**Pull request closed without merging**